README.md

```diff
@@ -102,22 +102,21 @@ themeSwitcher.addEventListener('change', () => {
 module.exports = {
   plugins: [
     require('postcss-dark-theme-class')({
-      darkClass: 'dark-theme',
-      lightClass: 'light-theme'
+      darkSelector: '.dark-theme',
+      lightSelector: '.light-theme'
     })
   ]
 }
 ```
 
-### `darkClass`
+### `darkSelector`
 
 Type: `string`. Default: `is-dark`.
 
-`html`’s class, which will switch dark theme.
+`css`'s valid selector, which will switch dark theme.
 
-
-### `lightClass`
+### `lightSelector`
 
 Type: `string`. Default: `is-light`.
 
-`html`’s class, which will switch light theme.
+`css`'s valid selector, which will switch light theme.
```

Comment:
`html` here means a `<html>` tag. You should use `CSS’s valid selector` if you are talking about the language.

Reply:
So uppercase `CSS`?

Reply:
Maybe it will be better to clarify that we add selectors to `<html>` tag?

```
CSS’s valid selector for `<html>` (which is alias for `:root`) …

Use `lightSelector: '[data-theme=light]' if you will switch theme by setting `<html data-theme="light">`
```

Reply:
yes, doing it.

Reply:
You do not need code here as well since it is not the example of some code:

```diff
- Any `CSS`'s valid selector
+ Any CSS's valid selector
```

Reply:
> CSS’s valid selector for `<html>` (alias for `:root`), which will switch dark theme.

Reply:
:+1: